packages/itwinui-css/src/table/table.scss

```diff
@@ -88,7 +88,7 @@
 
       .iui-table-sort,
       .iui-table-filter-button {
-        visibility: visible;
+        opacity: revert;
```

Comment:
Minor, but `opacity: 1` looks better to me.

Reply:
I made this change.